exporters/memory/src/in_memory_span_data.cc

```diff
@@ -0,0 +1,40 @@
+#include "opentelemetry/exporters/memory/in_memory_span_data.h"
+
+using opentelemetry::sdk::common::CircularBuffer;
+using opentelemetry::sdk::common::CircularBufferRange;
+using opentelemetry::sdk::common::AtomicUniquePtr;
+
+const size_t MAX_BUFFER_SIZE = 100;
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace memory
+{
+InMemorySpanData::InMemorySpanData() : spans_received_(MAX_BUFFER_SIZE) {}
```

Comment:
Let's add a constructor argument for the maximum buffer size, using MAX_BUFFER_SIZE as a default.

Reply:
Done